client/src/root/Logo.jsx

```diff
@@ -221,7 +221,6 @@ export default class Logo extends Component {
         <div style={styleTextWrapperMerged}>
           <a
             href="#"
-            title="One Stop Home"
```

Comment:
curious why we got rid of this, is it because the content of the <a> encapsulates the proper information alone? I'm guessing this was a web team thing?

Reply:
The webteam didn't like that the text of the link didn't match the title (screen readers only read the title part when it's set - it basically overrides the text value). Since we have the separate menu with the link labeled "Home", it was easiest to just remove the title so the text is the same for every user.